TypeCobol.Test/Misc/MatchingProgramEndTest.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Globalization;
+using System.IO;
+using System.Linq;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using TypeCobol.Compiler;
+using TypeCobol.Compiler.CodeElements;
+using TypeCobol.Compiler.CodeModel;
+using TypeCobol.Compiler.Nodes;
+using TypeCobol.Test.Utils;
+
+namespace TypeCobol.Test.Misc
+{
+    [TestClass]
+    public class MatchingProgramEndTest
+    {
+        /// <summary>
+        /// Issues #1162 #1844 parse "ProgramEnd.rdz.cbl" in Parser\Programs\Cobol85\
+        /// And check END PROGRAM is matched with its program declaration start
+        /// </summary>
+        [TestMethod]
+        [TestCategory("Parsing")]
+        [TestProperty("Time", "fast")]
+        public void CheckMatchingEndProgram()
```

Comment:
You can achieve the same result if you create a file "TypeCobol.Test/Parser/Programs/Cobol85/ProgramEnd-Nodes.txt"

Reply:
File deleted